Dreamcast: More info about Online services

EuropaYou opened the pull request:
At the moment I just tried to add general information. I will check the information in #169 to see if it is correct or not. I also saw some non-English sources while I was researching the topic and I will check these sources as well.

It is currently a draft, so don't worry if there are grammar mistakes in the article.

EuropaYou committed:
Update dreamcast.Rmd.md

EuropaYou committed:
DreamPhone

EuropaYou committed:
Divided the article into regions

EuropaYou committed:
Mention about broadband adapter

EuropaYou committed:
made lists bold

flipacholas commented:
I would focus on the main topic of this pull request for now (online services). I'll also need you to list the sources here that back the new information. Thanks

EuropaYou committed:
Some minor fixes

EuropaYou committed:
Update dreamcast.Rmd.md

EuropaYou committed:
Merge branch 'flipacholas:master' into master

EuropaYou committed:
extend American Service

https://www.ign.com/articles/2000/04/04/sega-changes-face-offers-dreamcast-for-free

EuropaYou committed:
SegaNet's 100k mark

https://www.ign.com/articles/2000/10/20/seganet-hits-the-100000-mark

EuropaYou committed:
some details

https://www.ign.com/articles/2000/04/05/everything-youve-ever-wanted-to-know-about-segas-internet-plans

EuropaYou committed:
Merge branch 'flipacholas:master' into master

EuropaYou committed:
Update dreamcast.Rmd.md

https://www.sega.jp/history/hard/dreamcast/

EuropaYou committed:
Added SegaNet account migration to EarthLink accounts

https://www.ign.com/articles/2001/07/30/seganet-links-up-with-earth

EuropaYou committed:
Merge branch 'flipacholas:master' into master

EuropaYou committed:
Added NetFront browser

Source: https://segaretro.org/Press_release:_1998-11-24:_ACCESS_Powers_Sega%27s_Dreamcast,_the_Next-generation_128-bit_TV-game_Machine,_with_Dream_Passport_--_for_Internet_Access_with_NetFront_and_Internet_Security_with_Secure_Socket_Layer

EuropaYou committed:
Merge branch 'flipacholas:master' into master

EuropaYou committed:
Added Taiwan

https://segaretro.org/Press_release:_2000-08-14:_GigaMedia_and_Sega.com_Asia_Ltd_to_Further_Partnership_Discussion_For_Building_On-line_Gaming_Services_in_Taiwan

EuropaYou committed:
Fixed indentation errors

EuropaYou committed:
Merge branch 'flipacholas:master' into master

EuropaYou committed:
Merge branch 'flipacholas:master' into master

EuropaYou committed:
Move Taiwan up

EuropaYou committed:
SegaNet 100k reach in 1month

https://segaretro.org/Press_release:_2000-10-19:_SegaNet_Tops_100,000_Members_in_Just_One_Month;_First_High_Speed_Online_Console_Gaming_Network_Draws_Huge_Community_With_High_Speed,_Five_Star_Content

EuropaYou committed:
Add NetFront features

https://segaretro.org/Press_release:_1998-11-24:_ACCESS_Powers_Sega%27s_Dreamcast,_the_Next-generation_128-bit_TV-game_Machine,_with_Dream_Passport_--_for_Internet_Access_with_NetFront_and_Internet_Security_with_Secure_Socket_Layer

EuropaYou committed:
Sega and AT&T

https://segaretro.org/Press_release:_1999-08-04:_Sega_and_AT%26T_Sign_Strategic_Network_Gaming_Alliance_for_Sega_Dreamcast

EuropaYou committed:
EU,AU,NZ registiration using Dreamkey and Dreamkey versions

https://segaretro.org/Dreamkey
https://segaretro.org/Dreamkey_Version_2.0
https://segaretro.org/Dreamkey_3.0

Commit:
Add Dreamcast Direct

https://segaretro.org/Press_release:_1999-04-23:_Dreamcast_Direct_Start

Commit:
Add Dream Passport

https://segaretro.org/Dream_Passport
https://segaretro.org/Dream_Passport_2
https://segaretro.org/Dream_Passport_3
https://segaretro.org/Dream_Passport_3/Hidden_content

Commit:
Add versions of DreamPass and broadband adapter reqs

https://segaretro.org/Sakura_Taisen_Dream_Passport_3
https://segaretro.org/Broadband_Passport
https://segaretro.org/Dream_Passport_2_for_LAN
https://segaretro.org/Hello_Kitty_Dream_Passport_2

Commit:
Delete duplicate Japan Online Service

Commit:
Improved Dreamarena

https://segaretro.org/Dreamarena

EuropaYou committed:
Merge branch 'flipacholas:master' into master
